src/task/task.service.ts

```diff
@@ -0,0 +1,106 @@
+import { faker } from '@faker-js/faker'
+import { Injectable } from '@nestjs/common'
+import { ConfigService } from '@nestjs/config'
+import { Cron, CronExpression } from '@nestjs/schedule'
+import { PrismaService } from '../database/prisma.service'
+
+@Injectable()
+export class TasksService {
+    constructor(private readonly prisma: PrismaService, private readonly config: ConfigService) {}
+
+    @Cron(CronExpression.EVERY_12_HOURS)
+    async createAuction() {
```

Comment:
Why not reuse `seedAuction()` from `database/seed`?
Also, I think it would be nice to provide some mechanism of configuring mode (Dev/Prod), because if that would be a real project, we won't want see such tasks in prod.